<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_cadastro_cliente</class>
 <widget class="QMainWindow" name="tela_cadastro_cliente">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>465</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>465</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastrar novo Cliente</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagens.qrc">
    <normaloff>:/img/logo_sex.png</normaloff>:/img/logo_sex.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gb_dados_cliente">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>790</width>
      <height>210</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="title">
     <string>Dados do Cliente</string>
    </property>
    <widget class="QLabel" name="lb_telefone">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>53</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Telefone:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_telefone">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>53</y>
       <width>217</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>22</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="maxVisibleItems">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="lb_nome">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>23</y>
       <width>39</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nome">
     <property name="geometry">
      <rect>
       <x>51</x>
       <y>23</y>
       <width>410</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>410</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>60</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_cpf">
     <property name="geometry">
      <rect>
       <x>647</x>
       <y>23</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>CPF:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_cpf">
     <property name="geometry">
      <rect>
       <x>675</x>
       <y>23</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="inputMask">
      <string>000.000.000-00; </string>
     </property>
     <property name="text">
      <string>..-</string>
     </property>
     <property name="maxLength">
      <number>14</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_rg">
     <property name="geometry">
      <rect>
       <x>487</x>
       <y>23</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>RG:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_rg">
     <property name="geometry">
      <rect>
       <x>508</x>
       <y>23</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_email">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>53</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>E-mail:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_email">
     <property name="geometry">
      <rect>
       <x>54</x>
       <y>53</y>
       <width>390</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>20</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>450</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="maxCount">
      <number>60</number>
     </property>
    </widget>
    <widget class="QLabel" name="lb_comentario">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Coment√°rio</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="te_comentario">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>770</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="tb_mais_email">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>49</y>
       <width>25</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="imagens.qrc">
       <normaloff>:/img/mais.png</normaloff>:/img/mais.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="tb_menos_email">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>63</y>
       <width>25</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="imagens.qrc">
       <normaloff>:/img/menos.png</normaloff>:/img/menos.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="tb_mais_telefone">
     <property name="geometry">
      <rect>
       <x>757</x>
       <y>49</y>
       <width>20</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="imagens.qrc">
       <normaloff>:/img/mais.png</normaloff>:/img/mais.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="tb_menos_telefone">
     <property name="geometry">
      <rect>
       <x>757</x>
       <y>63</y>
       <width>20</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="imagens.qrc">
       <normaloff>:/img/menos.png</normaloff>:/img/menos.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>12</width>
       <height>12</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_endereo">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>220</y>
      <width>790</width>
      <height>208</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="title">
     <string>Endere√ßo</string>
    </property>
    <widget class="QLineEdit" name="le_cep">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>23</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="inputMask">
      <string>00000-000; </string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="cursorPosition">
      <number>9</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_cep">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>23</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>CEP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_rua">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>23</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Rua:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_rua">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>23</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_numero">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>23</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>N√∫mero:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_numero">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>23</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_bairro">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>53</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Bairro:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_bairro">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>53</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>40</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_cidade">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>53</y>
       <width>46</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Cidade:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_cidade">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>53</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>40</number>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_estado">
     <property name="geometry">
      <rect>
       <x>555</x>
       <y>53</y>
       <width>46</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Estado:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_estado">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>53</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lb_ponto_refe">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>78</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Ponto de refer√™ncia:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="te_ponto_referencia">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>770</width>
       <height>98</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <zorder>lb_rua</zorder>
    <zorder>le_rua</zorder>
    <zorder>lb_numero</zorder>
    <zorder>le_numero</zorder>
    <zorder>lb_bairro</zorder>
    <zorder>le_bairro</zorder>
    <zorder>lb_cidade</zorder>
    <zorder>le_cidade</zorder>
    <zorder>lb_estado</zorder>
    <zorder>le_estado</zorder>
    <zorder>lb_ponto_refe</zorder>
    <zorder>te_ponto_referencia</zorder>
    <zorder>le_cep</zorder>
    <zorder>lb_cep</zorder>
   </widget>
   <widget class="QPushButton" name="btn_confirmar">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>435</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Confirmar</string>
    </property>
    <property name="icon">
     <iconset resource="imagens.qrc">
      <normaloff>:/img/1345902916_camera_test.png</normaloff>:/img/1345902916_camera_test.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancelar">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>435</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>24</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
    <property name="icon">
     <iconset resource="imagens.qrc">
      <normaloff>:/img/1345902747_button_cancel.png</normaloff>:/img/1345902747_button_cancel.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
